models/marts/core/fct_orders.sql

```diff
@@ -0,0 +1,42 @@
+with  orders as (
+
+    select * from {{ ref ('stg_orders')}}
+
+), stripe_payments as (
+
+    select * from {{ ref('stg_payments')}}
+
+
+), order_totals as (
+
+  select
+
+    order_id,
+
+    sum(case when payment_status = 'success' then amount_usd end) as amount_total_usd
+
+  from stripe_payments
+
+  group by 1
+
+), final as (
+
+    select
+
+        orders.order_id,
+
+        orders.customer_id,
+
+        orders.order_date,
+
+        coalesce(order_totals.amount_total_usd, 0) as amount_total_usd
+
+    from orders
+
+    left join order_totals
+
+      on orders.order_id = order_totals.order_id
```

Comment:
Totally up to you if you prefer explicit `on` or `using` in your SQL style. Having a consistent style across models is what's most important, and `dim_customers` currently has `using`. (I'm mostly just happy that both columns are named `order_id`.)